test/http-api/over-ipfs-api/bootstrap.js

```diff
@@ -28,6 +28,27 @@ module.exports = (ctl) => {
         })
       })
 
+      it('prevents duplicate inserts of bootstrap peers', () => {
+        return ctl
+          .bootstrap
+          .add(validIp4)
+          .then(res => {
+            expect(res).to.be.eql({ Peers: [validIp4] })
+            return ctl.bootstrap.add(validIp4)
+          })
+          .then((res) => {
+            expect(res).to.be.eql({ Peers: [validIp4] })
+            return ctl.bootstrap.list()
+          })
+          .then((res) => {
+            expect(res).to.exist()
+            const insertPosition = res.Peers.indexOf(validIp4)
+            expect(insertPosition).to.not.equal(-1)
+            const duplicatePosition = res.Peers.indexOf(validIp4, insertPosition + 1)
+            expect(duplicatePosition).to.equal(-1)
```

Comment:
That way, this test would also be protected against future changes to the default bootstrap list.